apps/codeforafrica/src/components/FooterMenu/index.js

```diff
@@ -0,0 +1,52 @@
+import { Link } from "@commons-ui/next";
+import PropTypes from "prop-types";
+import React from "react";
+
+import NavList from "@/codeforafrica/components/NavList";
+import NavListItem from "@/codeforafrica/components/NavListItem";
+
+function FooterMenu({ menu }) {
+  if (!menu?.length) {
+    return null;
+  }
+
+  return (
+    <NavList direction="column">
+      {menu.map((item) => (
+        <NavListItem key={item.label}>
+          <Link
+            href={item.href}
+            sx={{
+              color: "inherit",
+              background: "inherit",
+              textDecoration: "none",
+              fontWeight: 700,
+              fontSize: 23,
+              "&:hover, &:active, &:focus, &:focus-within": {
+                textDecoration: "none",
+                color: "inherit",
+              },
+            }}
```

Comment:
```suggestion
            color="inherit",
            underline="none",
            variant="h5",
            sx={{
              background: "inherit",
              "&:hover, &:active, &:focus, &:focus-within": {
                color: "inherit",
              },
            }}
```

`Link` (Being based on `Typography`) supports most of these customisations directly via props. And in footer, I think the font is always `h5`.